doc/langref.html.in

```diff
@@ -2861,6 +2861,36 @@ fn dump(args: anytype) void {
     expect(args.b);
     expect(args.s[0] == 'h');
     expect(args.s[1] == 'i');
+}
+      {#code_end#}
+      <p>
+      Anonymous structs without field names may be created, and are referred to as tuples.
+      </p>
+      <p>
+      They field names are given numbered names starting with "0",  which may be accessed with the special syntax @"0". Things inside @"" are always recognised as identifiers.
+      </p>
+      <p>
+      These also share some properties of arrays: they can be iterated over, indexed, used with the ++ and ** operators, and have a len field.
```

Comment:
Like arrays, tuples have a .len field, can be indexed, can be be iterated over and they work with the ++ and ** operators.

Reply:
They also can only be iterated over with `inline for`. (Is this a bug?)

Reply:
I don't think it's a bug. They are struct fields which are known at compile time and can also only be iterated over with `inline for`.